wiki/markdown/failedNotification.md

```diff
@@ -0,0 +1,17 @@
+# FailedNotification
+
+|Method|Endpoint             |
+|------|---------------------|
+|GET   |/failed-Notifications|
+
+
+## failed notifications
+    GET /failed-Notifications
+
+**Query parameters**:
+SubscriptionName=<Subscription Name/Subscription ID>
+
+Examples of this endpoint using curl
+
+    curl -X GET -H "Content-type: application/json" localhost:39835/failed-Notifications?SubscriptionName=Subscription_1
```

Comment:
lowercase the N

Reply:
doNe

Reply:
parameter should be subscriptionName with lower case S. I changed it in my last PR I believe.

Reply:
done